docs-next/components/ComingSoon.tsx

```diff
@@ -0,0 +1,42 @@
+/** @jsx jsx */
+import { jsx, Box, Stack, css } from '@keystone-ui/core';
+import { Button } from '@keystone-ui/button';
+import Link from 'next/link';
+
+export const ComingSoon = () => {
+  return (
+    <Box
+      padding="medium"
+      rounding="medium"
+      foreground="cyan900"
+      background="neutral100"
+      css={{ borderWidth: '1px' }}
+    >
+      <Stack
+        css={{
+          width: '100%',
+          justifyContent: 'space-between',
+        }}
+        gap="small"
+        align="center"
+        across
+      >
+        <p css={{ margin: '0 !important ' }}>
+          Coming Soon...Visit our roadmap for more inforamtion
+        </p>
+        <Link href="/roadmap" passHref>
+          <Button
+            css={css`
```

Comment:
we have to do this because once again `tailwind`'s prose class applies styles of the highest specificity to elements, in the tree. 

This can be removed once we opt to rip out the prose class from our docs